component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentBottom.kt

```diff
@@ -12,14 +12,14 @@ class ResultInvestmentBottom @JvmOverloads constructor(
     context: Context,
     attributeSet: AttributeSet? = null,
     defStyleAttr: Int = 0
-): LinearLayout(context, attributeSet, defStyleAttr) {
+) : LinearLayout(context, attributeSet, defStyleAttr) {
 
-    private val viewBinding:InvestmentResultBottomBinding by lazy {
+    private val viewBinding: InvestmentResultBottomBinding by lazy {
         InvestmentResultBottomBinding.inflate(LayoutInflater.from(context))
     }
 
     init {
-       addView(viewBinding.root, LayoutParams(LayoutParams.MATCH_PARENT,  LayoutParams.MATCH_PARENT))
+       addView(viewBinding.root, LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT))
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fioalpha/CaseStudy/blob/6ce9cd99c0c1ee323fc1eba2aae62b43356b0111/component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentBottom.kt#L22"></span>Unexpected indentation (7) (it should be 8) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>